processes/security_team_onboarding.md

```diff
@@ -25,7 +25,7 @@ Patiently wait for feedback and +1’s from the team and a reasonable time to en
 Once acceptance has been acquired by group members, the following should take place to setup the new member:
 
 * Confirm that the new member has read, understands, and agrees to uphold the
-  [Code of Conduct](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md).
+  [Code of Conduct](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md). The work of triaging vulnerabilities at the Security WG is about handling sensitive matters with open source maintainers and communication should be handled with care. If in doubt on how to communicate reach out to the Moderation WG team and ask for guidance. As a member of the Security WG please also be aware of public statements your make and interactions.
```

Comment:
The moderation group can help, but I  think asking on the security-wg slack is a good first step, too. Or technology aside, its good to work together with other people on the sec-wg before going the next step and involving moderation team.

Reply:
Understand where you're coming from and hopefully it should go without saying that discussing it with friends on the wg slack is a good first step. With that said, in the matters of communicating on public forums the sec wg may not be well trained on it even if they have good intentions - therefore outlining here that the Moderation team is a helpful channel is important to mention in my opinion.

Reply:
Will resolve the conversation and we can pick it up in a new PR if needed